docs/paper/admin/reference/system-properties.md

```diff
@@ -215,3 +215,8 @@ It also remaps plugin CB calls to remove the version information.
 
 - **default**: `false`
 - **description**: Disables plugin remapping introduced in 1.20.5 and everything involving automatic plugin conversion by commodore. For more information see the [userdev](../../dev/getting-started/userdev.mdx#1205-and-beyond) documentation and the official [announcement](https://discord.com/channels/289587909051416579/976631292747735080/1232740079097876570).
+
+#### paper.startSparkBackgroundProfilerImmediately
+
+- **default**: `false`
+- **description**: Starts the background profiler as soon as possible when set to true. Otherwise the background profiler starts once the server has finished starting up (when the `Done (X.XXXs)! For help, type "help"` message is sent in console).
```

Comment:
```suggestion
- **description**: Enables the spark profiler as soon as possible when set to true. Otherwise enables once the server has finished starting up (when the `Done (X.XXXs)! For help, type "help"` message is sent in console).
```